### PR TITLE
Add branch 18.0 (no default branch change)

### PR DESCRIPTION
**Pull request by simahawk**
Possible replacement for https://github.com/OCA/repo-maintainer-conf/pull/36

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: conf/repo/social.yml [hunk 1]
```diff
@@ -10,17 +20,9 @@ social:
     - "15.0"
     - "16.0"
     - "17.0"
+    - "18.0"
   category: Social
   default_branch: "14.0"
-  name: Addons concerning odoo's social erp features and messaging in general
```
Comment (etobella) [inline conf/repo/social.yml:15]: Why did you remove this?

Comment (etobella) [reply conf/repo/social.yml:15]: I see, it was moved automatically

**Review by etobella — APPROVED** [review]

**Comment by simahawk** [comment]
@sbidoul any blocker for the merge?

**Comment by sbidoul** [comment]
I was wondering if we could test on a couple more repos, to save me a mass update in case something is missing in the template, but otherwise, good for me.

**Comment by simahawk** [comment]
@sbidoul we have some WIP on v18 already. I'll open another PR w/ a shortlist of repos (eg: queue, connector, etc)

**Comment by simahawk** [comment]
My subset here https://github.com/OCA/repo-maintainer-conf/pull/43

**Comment by simahawk** [comment]
IMO this is working pretty well. I don't see any other reason to wait. Do you?

**Comment by simahawk** [comment]
@sbidoul approved :)

**Comment by simahawk** [comment]
Looks like we have an issue https://github.com/OCA/repo-maintainer-conf/issues/44
